src/traits.rs

```diff
@@ -958,6 +958,16 @@ pub trait SystemExt: Sized + Debug + Default {
     /// println!("OS version: {:?}", s.get_version());
     /// ```
     fn get_version(&self) -> Option<String>;
+
+    /// Returns the system hostname based off DNS
+    ///
```

Comment:
Could you add `/// **Important**: this information is computed every time this function is called.` like in `fn get_name` please?

Reply:
Done 👍